### ISSUE TITLE
Warnings with Ruby v2.7.1

### ISSUE THREAD

**Issue by dandunckelman** [zhandao/zero-rails_openapi#66]
We're working toward updating our Rails project to Ruby v2.7.1, and when our project starts, we get these warnings:

```
/usr/local/bundle/gems/zero-rails_openapi-2.1.4/lib/oas_objs/request_body_obj.rb:21: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
/usr/local/bundle/gems/zero-rails_openapi-2.1.4/lib/oas_objs/media_type_obj.rb:12: warning: The called method `initialize' is defined here
/usr/local/bundle/gems/zero-rails_openapi-2.1.4/lib/oas_objs/response_obj.rb:23: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
/usr/local/bundle/gems/zero-rails_openapi-2.1.4/lib/oas_objs/media_type_obj.rb:12: warning: The called method `initialize' is defined here
/usr/local/bundle/gems/zero-rails_openapi-2.1.4/lib/open_api/dsl/helpers.rb:34: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
/usr/local/bundle/gems/zero-rails_openapi-2.1.4/lib/open_api/dsl/helpers.rb:27: warning: The called method `_combined_schema' is defined here
```

You can see the deprecation details here: https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/

**Comment by zhandao** [comment]
Thank you. I'll take the time to fix it.

**Comment by zhandao** [comment]
Sorry for the delay.

This fix was released in version `2.1.5`.
